NEWS

```diff
@@ -7,6 +7,7 @@
 
   Major changes between OpenSSL 1.1.0j and OpenSSL 1.1.0k [under development]
 
+      o Fix segfault in `openssl speed'
```

Comment:
Should this be placed in CHANGES file?

Reply:
This should go in CHANGES, if anything (I'd encourage it, CHANGES is greatly underused)

Reply:
I wouldn't put it in either CHANGES or NEWS personally.